CONTRIBUTING.md

```diff
@@ -304,8 +304,73 @@ We are excited to help you contribute to any of the topics that you would like t
 
 When in doubt, you can reach out to current project lead(s):
 
-| Name            | GitHub | Twitter |
-|:----------------|:-------|:--------|
-| Quincy Larson | [`@QuincyLarson`](https://github.com/QuincyLarson) | [`@ossia`](https://twitter.com/ossia)|
+| Name            | GitHub | Role |
```

Comment:
@allella I would extract this into README.md

Reply:
I'm going to merge this into CONTRIBUTING.md and we'll consider moving to README